### PR TITLE
Exclude bundle.*.js and stats.json with .gitignore

### PR DESCRIPTION
**Pull request by obrientimothya**
Added compiled bundle.*.js and local stats.json files to .gignore to ensure they are not accidentally committed, if present.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,9 @@ bower_components
 Thumbs.db
 
 .env
+
+# Bundled scripts
+website/assets/js/bundle.*.js
+
+# Stats
+stats.json
```
Comment (sahat) [inline .gitignore]: stats.json has been removed entirely

File: .gitignore [hunk 2]
```diff
@@ -36,3 +36,9 @@ bower_components
 Thumbs.db
 
 .env
+
+# Bundled scripts
+website/assets/js/bundle.*.js
```
Comment (sahat) [inline .gitignore]: this could be updated to just bundle.js, since chunkhash is no longer generated.

**Comment by obrientimothya** [comment]
Thanks @sahat 
- removed stats.json and generalized the wildcard for bundles to handle with/without chunkhash.

**Comment by sahat** [comment]
Looks good!

**Pull request merged**